src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -117,6 +121,27 @@ internal class WorkloadInformationProvider : IWorkloadInformationProvider
             }
         }
 
+        //
+        public List<int> GetPortsToIgnoreFromAnnotations(V1ServiceList serviceNameSpace) {
+            var portsToIgnore = new List<int>();
+            foreach(var item in serviceNameSpace.Items) {
+                if (item.Metadata != null && item.Metadata.Annotations != null) {
+                    if (item.Metadata.Annotations.TryGetValue("bridgetokubernetes/ignore-ports", out var ports)) {

```

Comment:
💡 since we are using `split` I am guessing it guarantee `string type` so lets explicitly mention that, `out string ports`?

Reply:
I have simplified the func which avoids the use of split.